pkg/apis/configuration/validation/policy.go

```diff
@@ -56,7 +56,7 @@ func validatePolicySpec(spec *v1.PolicySpec, fieldPath *field.Path, isPlus, enab
 	if spec.OIDC != nil {
 		if !enablePreviewPolicies {
```

Comment:
I would rename this to reflect the new OIDC flag

Reply:
Done - see below comment